bundles/org.openhab.binding.caddx/README.md

```diff
@@ -0,0 +1,246 @@
+| zone_dialer_delay                                | Contact   | Configuration       | Dialer delay                               |
+| zone_swinger_shutdown                            | Contact   | Configuration       | Swinger shutdown                           |
+| zone_restorable                                  | Contact   | Configuration       | Restorable                                 |
+| zone_listen_in                                   | Contact   | Configuration       | Listen in                                  |
+| zone_faulted                                     | Contact   | Zone Condition      | Faulted (or delayed trip)                  |
+| zone_tampered                                    | Contact   | Zone Condition      | Tampered                                   |
+| zone_trouble                                     | Contact   | Zone Condition      | Trouble                                    |
+| zone_bypassed                                    | Switch    | Zone Condition      | Bypassed                                   |
+| zone_inhibited                                   | Contact   | Zone Condition      | Inhibited (force armed)                    |
+| zone_low_battery                                 | Contact   | Zone Condition      | Low battery                                |
+| zone_loss_of_supervision                         | Contact   | Zone Condition      | Loss of supervision                        |
+| zone_alarm_memory                                | Contact   | Zone Condition      | Alarm memory                               |
+| zone_bypass_memory                               | Contact   | Zone Condition      | Bypass memory                              |
+
+## Full Example
+
+The following is an example of a things file (caddx.things):
+
+```
+Bridge caddx:bridge:thebridge  "Bridge"                   [ protocol="Binary", serialPort="/dev/ttyUSB0", baud=38400 ] {
+    Thing partition partition1 "Groundfloor alarm"        [ partitionNumber=1 ]
+    Thing zone      zone1      "Livingroom motion sensor" [ zoneNumber=1 ]
+    Thing zone      zone2      "Bedroom motion sensor"    [ zoneNumber=2 ]
+    Thing zone      zone3      "Guestroom motion sensor"  [ zoneNumber=3 ]
+    Thing zone      zone4      "Livingroom window"        [ zoneNumber=4 ]
+    Thing zone      zone5      "Bedroom window"           [ zoneNumber=5 ]
+    Thing zone      zone6      "Guestroom window"         [ zoneNumber=6 ]
+}
+```
+
+The following is an example of an item file (caddx.items):
+
+```
+Group:Switch:OR(ON, OFF)        MotionSensors   "Motion Sensors"   <motion>   ["MotionDetector"]
+Group:Switch:OR(ON, OFF)        Windows         "Windows"          <window>   ["Window"]
+
+Switch    Bedroom_Motion        "Motion Sensor [%s]"  <motion>       (MotionSensors)  { channel="caddx:zone:thebridge:zone1:zone_faulted" }
```

Comment:
Since you changed them all to contacts wouldn't this need to changed too?

Reply:
I changed all the Switch items to Contact items based on the concept to use a Contact for output only and to use a Switch for input as well.

It does not feel correct.
e.g. Zone Faulted? [On/Off] feels more natural than [Open/Closed]
There is no channel like zone_status to model it as a Contact.

I am more tempted to revert all items back to Switch.
Any suggestion?

Reply:
My feeling is that Contact items should be reserved for sensors. Most alarm system use contact sensor for doors/windows and other zones so it seems natural to use the Contact item to reflect their state in openhab. But given that on/off translates better to yes/no than open/close does I can understand why you would want to use a switch instead.

My suggestion is to just use the item type that makes sense to you. No you don't have to make everything a Contact type, though you should still make the channels that relate to the open/close states of windows/doors use Contact types.

Reply:
Changed everything back to Switch except the zone_faulted channel.